--- Makefile
@@ -49,15 +49,7 @@
 		scdoc < "$$f" | gzip > "$$target"; \
 	done
 install:
-	@if [ -e /run/systemd/system ]; then \
-		mkdir -p $(DESTDIR)$(PREFIX)/lib/systemd/system/; \
-		install -Dm644 keyd.service $(DESTDIR)$(PREFIX)/lib/systemd/system/keyd.service; \
-	else \
-		echo "NOTE: systemd not found, you will need to manually add keyd to your system's init process."; \
-	fi
-
 	@if [ "$(VKBD)" = "usb-gadget" ]; then \
-		install -Dm644 src/vkbd/usb-gadget.service $(DESTDIR)$(PREFIX)/lib/systemd/system/keyd-usb-gadget.service; \
 		install -Dm755 src/vkbd/usb-gadget.sh $(DESTDIR)$(PREFIX)/bin/keyd-usb-gadget.sh; \
 	fi
 
@@ -80,7 +72,8 @@
 
 uninstall:
 	-groupdel keyd
-	rm -rf $(DESTDIR)$(PREFIX)/lib/systemd/system/keyd.service \
+	rm -rf \
+		$(DESTDIR)$(PREFIX)/etc/init.d/keyd \
 		$(DESTDIR)$(PREFIX)/bin/keyd \
 		$(DESTDIR)$(PREFIX)/bin/keyd-application-mapper \
 		$(DESTDIR)$(PREFIX)/share/doc/keyd/ \
